bin/Makefile.am

```diff
@@ -34,14 +34,15 @@ dist_bin_SCRIPTS = ch-build \
 ## Python scripts - need text processing
 
 bin_SCRIPTS = ch-run-oci
-EXTRA_SCRIPTS = ch-grow
-if ENABLE_CH_GROW
-bin_SCRIPTS += ch-grow
+EXTRA_SCRIPTS = ch-image
+if ENABLE_CH_IMAGE
+bin_SCRIPTS += ch-image ch-grow
 endif
-EXTRA_DIST = ch-grow.py.in ch-run-oci.py.in
-CLEANFILES = $(bin_SCRIPTS)
+EXTRA_DIST = ch-grow.py.in ch-image.py.in ch-run-oci.py.in
+CLEANFILES = $(bin_SCRIPTS) ch-grow
 
 ch-grow: ch-grow.py.in
```

Comment:
and here?

Reply:
We need `ch-grow` to be non-exectuable if the requirements for `ch-image` aren't met; otherwise `exectuables sane` will fail Travis `MINIMUM DEPS` tests. It calls `ch-grow` with zero arguments and since `ch-image` doesn't exist we get a fatal error that Travis isn't expecting.

I thought about adding logic to the `Makefile` to change executable bits but it seemed simpler to use existing logic above.